frontend/src/components/page_content/page-content-template.js

```diff
@@ -30,6 +30,14 @@ class PageContentTemplate extends React.Component {
 
     this.next = this.next.bind(this);
     this.prev = this.prev.bind(this);
+    this.save = this.save.bind(this);
+  }
+
+  save() {
+    //Save form and move to view plan
+    if (this.saveLabel === "View Plan") {
```

Comment:
i think you can just put this if statement in "next()" and then set the onSave={this.next}. then you can have else increment step. you shouldn't need save() or this.saveClick